src/docfx/docfx.csproj

```
@@ -41,4 +41,19 @@
     <ProjectReference Include="../Microsoft.Docs.Template/Microsoft.Docs.Template.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

remove this, it is already copied

---

what do you mean it's already copied?

---

My mistake, I thought we copied everything under `data` link we did for test project, so use:
```suggestion
   <None Include="data/**/*" CopyToOutputDirectory="PreserveNewest" />
```